src/output.rs

```diff
@@ -17,10 +18,14 @@ pub enum Argument {
 }
 
 impl FilterPrototype {
-    pub fn new(name: &str, arguments: Vec<Argument>) -> FilterPrototype {
+    pub fn new(name: &str,
+               arguments: Vec<Argument>,
+               named_arguments: Vec<(String, Argument)>)
```

Comment:
I was hesitant between `Vec<(String, Argument)>` and `Map<(String, Argument)>`.
Suggestions?